stdlib/2and3/zipfile.pyi

```diff
@@ -48,6 +50,7 @@ class ZipFile:
     def setpassword(self, pwd: bytes) -> None: ...
     def read(self, name: _SZI, pwd: Optional[bytes] = ...) -> bytes: ...
     def testzip(self) -> Optional[str]: ...
+    def _writecheck(self, zinfo: ZipInfo) -> None: ...
```

Comment:
I think that typically we don't include private methods like this in typeshed stubs.

Otherwise looks good.

Reply:
Meaning that if you _were_ going to use this method, it's better to have a `# type: ignore`?

Reply:
Yes. Private methods aren't stable and can change without notice, so using one could well break your code on a different Python version.